src/components/Calculator.jsx

```diff
@@ -0,0 +1,43 @@
+import Button from './shared/Button/Button';
+import './style.css';
+
+const Calculator = () => (
```

Comment:
- [ ] the `Calculator` component should be class-based according to the requirements